src/uct/ib/rdmacm/rdmacm_cm_ep.c

```diff
@@ -372,12 +379,18 @@ ucs_status_t uct_rdmacm_cm_ep_disconnect(uct_ep_h ep, unsigned flags)
 
     UCS_ASYNC_BLOCK(uct_rdmacm_cm_ep_get_async(cep));
     if (ucs_unlikely(cep->flags & UCT_RDMACM_CM_EP_FAILED)) {
-        ucs_error("%s: id=%p to peer %s",
-                  uct_rdmacm_cm_ep_str(cep, ep_str, UCT_RDMACM_EP_STRING_LEN),
-                  cep->id, ucs_sockaddr_str(rdma_get_peer_addr(cep->id),
-                                            ip_port_str,
-                                            UCS_SOCKADDR_STRING_LEN));
-        status = cep->status;
+        if (cep->status == UCS_ERR_CONNECTION_RESET) {
+            /* the ep was failed during wireup, remote side is not connected
+             * or destroyed */
+            status = UCS_OK;
```

Comment:
why UCS_OK? caller should be issue disconnect on a failed endpoint

Reply:
if the EP failed during wireup then should not

Reply:
but in UCP, you remove LOCAL_CONNECTED flag, in order to avoid calling disconnect on a failed endpoint per your comment https://github.com/openucx/ucx/pull/5166/files#r427789163). So why need to avoid error report on UCT side?

Reply:
this case is caused by uct_flush specific, it's possible when UCP do not send anything on TL lanes flushed empty lanes and interrupts wireup by destroying all lanes including CM lane